bin/light-base/src/sync_service/standalone.rs

```diff
@@ -94,16 +94,47 @@ pub(super) async fn start_standalone_chain<TPlat: Platform>(
     let mut from_network_service = from_network_service.fuse();
 
     // Main loop of the syncing logic.
+    //
+    // This loop contains some CPU-heavy operations (e.g. verifying justifications and warp sync
```

Comment:
maybe stupid question, but would it make sense to use threads to separate verification and foreground ops? That way we won't have to play scheduler ourselves

Reply:
Wasm is always single threaded. Additionally, the complexity of everything would probably be multiplied by 10 if you introduced concurrency in this code.